### PR TITLE
Update gems

### PR DESCRIPTION
**Pull request by dometto**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -1,10 +1,10 @@
 source 'https://rubygems.org'
-
+gem 'gollum-lib', :path => '~/Code/gollum_project/gollum-lib/'
```
Comment (bartkamphorst) [inline Gemfile:2]: Please remove this local `:path` to gollum-lib from the Gemfile.

File: gollum.gemspec [hunk 2]
```diff
@@ -23,18 +23,18 @@ Gem::Specification.new do |s|
   s.rdoc_options = ['--charset=UTF-8']
   s.extra_rdoc_files = %w[README.md LICENSE]
 
-  s.add_dependency 'gollum-lib', '~> 5.0'
+#  s.add_dependency 'gollum-lib', '~> 5.0'
```
Comment (bartkamphorst) [inline gollum.gemspec:26]: Why is this line commented out?

File: gollum.gemspec [hunk 3]
```diff
@@ -23,18 +23,18 @@ Gem::Specification.new do |s|
...
   s.add_dependency 'kramdown', '~> 2.3'
   s.add_dependency 'kramdown-parser-gfm', '~> 1.1.0'
   s.add_dependency 'sinatra', '~> 2.0'
   s.add_dependency 'sinatra-contrib', '~> 2.0'
-  s.add_dependency 'mustache', ['>= 0.99.5', '< 1.0.0']
+  s.add_dependency 'mustache-sinatra', '~> 1.0'
```
Comment (bartkamphorst) [inline gollum.gemspec:31]: Did this gem change names?

Comment (dometto) [reply gollum.gemspec:31]: Not exactly, but the Sinatra helpers that we were using were abstracted to a separated gem (also maintained by the mustache org), which in turn requires `mustache`.

File: gollum.gemspec (hunk 2 above)
Comment (dometto) [reply gollum.gemspec:26]: That's part of requiring my local copy of gollum-lib (for the purpose of testing the octicons). Thanks for catching these!

Comment (bartkamphorst) [reply gollum.gemspec:26]: I figured as much! :)

File: gollum.gemspec (hunk 3 above)
Comment (bartkamphorst) [reply gollum.gemspec:31]: 👍

**Pull request merged**